### ISSUE TITLE
Opening System Dynamics Models Fails Silently

### ISSUE THREAD

**Issue by wrand** [NetLogo/LevelSpace#110]
In the MOOC someone asked me about using LevelSpace with System Dynamics Models. I didn't expect it to work, but it seems to fail in a weird way. To reproduce:

1.  Opened Parent Model Example
2. Clicked Setup and then Create a Child Model
3. Chose Wolf-Sheep Predation (System Dynamics) from the Models Library
---nothing is displayed and no window pops up
4. ls:models is [0]
5. However, if you go to the LevelSpace menu bar and Click Edit Open Models, Wolf-Sheep is there, and I can bring up its code

It seems like either it shouldn't load at all and present an error message or load correctly.

**Comment by qiemem** [comment]
Good catch! I *think* I can probably get system dynamics models to work, but I'm not sure. I can definitely see how getting them to work would be useful: hybrid models seem to pretty common among those doing multi-level modeling. If I can't get it to work, though, I'll make sure it errors clearly and document it appropriately.
